mailinglogger/MailingLogger.py

```diff
@@ -112,6 +113,14 @@ def emit(self,record):
             email['Message-ID']=make_msgid('MailingLogger')
             smtp = smtplib.SMTP(self.mailhost, self.mailport)
             if self.username and self.password:
+                if self.secure is not None:
+                    # there is a secure option
+                    if self.secure:
+                        secure = [item for item in self.secure.split(' ') if item]

```

Comment:
This just feels a bit weird... Why is 'secure' being used to store multiple parameters? Why is None suddenly a magic value?